presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -1376,7 +1379,17 @@ protected void doTestMismatchSchemaTable(
             int filterCount = afterFilters.size();
             for (int i = 0; i < filterCount; i++) {
                 RowExpression predicate = afterFilters.get(i);
-                ConnectorTableLayoutHandle layoutHandle = metadata.pushdownFilter(session, tableHandle, predicate, Optional.empty()).getLayout().getHandle();
+                ConnectorTableLayoutHandle layoutHandle = pushdownFilter(
+                        session,
+                        metadata,
+                        transaction.getMetastore(""), // function does not use param in all implementations?
```

Comment:
I dont think we intended this Transaction interface to have a param for getMetastore - I dont see it being used anywhere (even in the internal codebase) am I missing something?

Reply:
@sachdevs Let's remove the parameter then. Just check prism connector first.